bower.json

```diff
@@ -6,34 +6,34 @@
     "build/ng-admin.min.js"
   ],
   "dependencies": {
-    "angular": "~1.3.1",
+    "angular": "~1.3.1"
+  },
+  "devDependencies": {
```

Comment:
We'll have to decide which is the default, between ng-admin only and full:

1. if ng-admin only, all dependencies below must be `dependencies`, because they are required to work with. Developers will need to include them (it can be done automatically in dev with [wiredep](https://github.com/taptapship/wiredep), that's what Yeoman does)
2. if full, all dependencies below must be `devDependencies`

I think 2 must be for testing purposes only, the target should be 1, and letting the developers do their own build phase.